keyboards/bandominedoni/bandominedoni.c

```diff
@@ -16,58 +16,6 @@
 
 #include "bandominedoni.h"
 
-#ifdef RGB_MATRIX_ENABLE
```

Comment:
Do these migrations produce the same output?

Reply:
It did the previous time I checked it, but just now it didn't. :man_shrugging: Reverting.